src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -12,6 +12,14 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\samples\\AppConfiguration\\Generated"
     },
+    "azure-parameter-grouping": {
```

Comment:
Your new powershell version reorders parameters and would cause a failure on CI. You might need to downgrade until devops gets it's powershell updated.